src/Orleans/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1"
+    "Newtonsoft.Json": "7.0.1",
+    "Microsoft.Extensions.DependencyInjection": "1.0.0"
```

Comment:
Hmm, not sure if it's avoidable (considering that we want to enhance how to configure Orleans by using DI, similar to AspNet MVC), but since this is currently the only binary that EVERY project needs to reference (including grain interfaces), I'm at least initially reluctant to add this dependency without considering alternative approaches.